### DISCUSSION TITLE
How to adjust the width of the sample column in the Feature Details panel

### DISCUSSION THREAD

**Discussion by yaanlpc** [GMOD/jbrowse-components#3469] [General] [answered]
As the sample ids contain useful informatio
![Annotation 2023-01-23 145800](https://user-images.githubusercontent.com/13814897/214189731-90386be9-1dde-429f-bba6-a036d22d6344.png)
n, thus it would be helpful if the column width can be adjusted as per requirement.

**Comment by cmdcolin** [comment, marked as answer]
thanks for reporting. I started making a little method for resizing columns on our data grids in this PR (https://github.com/GMOD/jbrowse-components/pull/3304), screenshot of how it looks:

 (the grey header allows clicking and dragging column widths using the black tick marks) 

![](https://user-images.githubusercontent.com/6511937/214151815-d2250664-1fec-4ff2-bb3e-84578ab2e935.png)

also, what jbrowse version are you using? it should even in current versions try to auto-size the columns better

**Reply by yaanlpc** [reply]
we are using the latest version 2.33, so far, as you can see from the snapshot, the tick scale is not black but grey, thus I am unable to manually adjust it wider to show full information of that sample.

**Reply by cmdcolin** [reply]
gotcha, thanks for confirming. the grey bar would be an upcoming feature, and the auto-sizing of the columns doesn't exist on the vcf sample grid like i thought it did, so both of these will get some fixes soon!

**Reply by cmdcolin** [reply]
@yaanlpc if you want to try a quick fix you can use `jbrowse create --branch variant_sample_grid newdir` to download a branch that autosizes and has the resizer

**Reply by cmdcolin** [reply]
we will try to incorporate that fix into an official release soon

**Reply by yaanlpc** [reply]
thanks Colin, looking forward to this feature.

**Reply by cmdcolin** [reply]
this is now out in v2.3.4, thanks for reporting this :)
